docs/graphx-programming-guide.md

```diff
@@ -708,7 +708,9 @@ messages remaining.
 > messaging function.  These constraints allow additional optimization within GraphX.
 
 The following is the type signature of the [Pregel operator][GraphOps.pregel] as well as a *sketch*
-of its implementation (note calls to graph.cache have been removed):
+of its implementation (note: to avoid stackOverflowError due to long lineage chains, graph and 
```

Comment:
OK. I am thinking in the original graphx-programming-guide, only cache is called. After we add do checkpointing and give notes we do that in the sketch, do we need to show the information in the sketch. If there is an agreement on remove all references to checkpoint, I will revert the changes.